.github/workflows/therock-ci-linux.yml

```
@@ -70,21 +68,25 @@ jobs:
       - name: Fetch sources
         timeout-minutes: 30
         run: |
-          ./TheRock/build_tools/fetch_sources.py --jobs 12 --no-include-math-libs
+          ./TheRock/build_tools/fetch_sources.py --jobs 12 --no-include-rocm-systems --no-include-rocm-libraries --no-include-ml-frameworks
 
       - name: Install python deps
         run: |
           pip install -r TheRock/requirements.txt
           pip freeze
 
+      - name: Patch rocm-systems
+        run: |
+          git -c user.name="therockbot" -c "user.email=therockbot@amd.com" am --whitespace=nowarn ./TheRock/patches/amd-mainline/rocm-systems/*.patch
+
       - name: Configure Projects
         env:
           amdgpu_families: ${{ env.AMDGPU_FAMILIES }}
           package_version: ADHOCBUILD
-          extra_cmake_options: "-DTHEROCK_USE_EXTERNAL_ROCM_LIBRARIES=ON -DTHEROCK_ROCM_LIBRARIES_SOURCE_DIR=../ ${{ inputs.cmake_options }}"
+          extra_cmake_options: "-DTHEROCK_USE_EXTERNAL_ROCM_SYSTEMS=ON -DTHEROCK_ROCM_SYSTEMS_SOURCE_DIR=${{ github.workspace }} ${{ inputs.cmake_options }}"
```

Looking into the logs, `rocm-systems` seems to be cloned to `'/__w/rocm-systems/rocm-systems'` whereas I see `-DTHEROCK_ROCM_SYSTEMS_SOURCE_DIR=/home/runner/_work/rocm-systems/rocm-systems`. Seems the checkout does not match `github.workspace` for whatever reasons.

This is the reason for not being able to find the HIP version file.
```suggestion
          extra_cmake_options: "-DTHEROCK_USE_EXTERNAL_ROCM_SYSTEMS=ON -DTHEROCK_ROCM_SYSTEMS_SOURCE_DIR=${GITHUB_WORKSPACE} ${{ inputs.cmake_options }}"
```
instead, see https://github.com/actions/checkout/issues/785 as this runs in a container.